lua/entities/ent_jack_sleepingbag.lua

```diff
@@ -0,0 +1,86 @@
+ENT.Type 			= "anim"
+ENT.PrintName		= "Sleeping Bag"
+ENT.Author			= "Basipek"
+ENT.Category			= "JMod - Misc"
+ENT.Spawnable			= true
+ENT.AdminSpawnable		= true
+
+
+if (CLIENT) then
+	function ENT:Draw()
+		self.Entity:DrawModel()
+	end
+elseif (SERVER) then
+
+	function ENT:Initialize()
+		self.Entity:SetModel("models/props_equipment/sleeping_bag1.mdl")
```

Comment:
if this comes from Left 4 Dead, then we need to port this model into JMod. We can't rely on people having JMod-Legacy installed just for this to work. JMod-Legacy is basically just an optional DLC for JMod. Gotta get both the model and the material in here